src/styles/util/variables.scss

```diff
@@ -0,0 +1,4 @@
+$card-width: 200px;
+$card-margin-r: 48px;
+$container-padding: 40px;
+$container-width: 4* $card-width + 3 * $card-margin-r + 2 * $container-padding;
```

Comment:
Let's make a function instead of variable to calculate the width of the container.